assets/sass/utilities/variables/_modal-var.scss

```diff
@@ -13,7 +13,7 @@
 }
 
 /// Define thin border radius.
-$border-radius-thin: 1px;
+$border-radius-thin: rem(1);
```

Comment:
Border radius should be `px` according to our UoM graphic. Do we need to update the theme or update the graphic?